tools/cldr-code/src/main/java/org/unicode/cldr/tool/CLDRFileTransformer.java

```diff
@@ -334,6 +334,17 @@ private String transformValue(
         return transliterated;
     }
 
+    private String transliterateByLines(Transliterator transliterator, String value) {
+        StringBuilder transliterated = new StringBuilder();
+        for (final String valueLine : value.split("[\r\n]")) {
+            if (transliterated.length() > 0) {
+                transliterated.append("\n");
+            }
+            transliterated.append(transliterator.transliterate(valueLine));
```

Comment:
It seems like this should also add a "\n", or am I missing something?

Reply:
Doesn't need a new line at the very end. The cdata emitting code puts a newline before the cdata end.